### PR TITLE
Add keyboard shortcut note

### PR DESCRIPTION
**Pull request by bamurtaugh**
### Why:

Addresses https://github.com/github/feedback/discussions/6162

### What's being changed:

Adds note about `.` potentially being inaccessible depending on user's local keyboard layout.

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@bamurtaugh 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by masumh — COMMENTED** [review]
Please help me my problem automatic adjust

**Review by masumh — COMMENTED** [review]
Please help me my problem automatic adjust

**Comment by masumh** [comment]
Please help me my problem automatic adjust

File: content/codespaces/the-githubdev-web-based-editor.md [hunk 1]
```diff
@@ -105,3 +105,4 @@ If you have issues opening the {% data variables.product.prodname_serverless %},
 
 - The {% data variables.product.prodname_serverless %} is currently supported in Chrome (and various other Chromium-based browsers), Edge, Firefox, and Safari. We recommend that you use the latest versions of these browsers.
 - Some keybindings may not work, depending on the browser you are using. These keybinding limitations are documented in the "[Known limitations and adaptations](https://code.visualstudio.com/docs/remote/codespaces#_known-limitations-and-adaptations)" section of the {% data variables.product.prodname_vscode %} documentation.
+- `.` may not work to open the {% data variables.product.prodname_serverless %} according to your local keyboard layout. In that case, you can open any {% data variables.product.prodname_dotcom %} repository in the {% data variables.product.prodname_serverless %} by changing the URL from "github.com" to "github.dev".
```
Comment (skedwards88) [inline content/codespaces/the-githubdev-web-based-editor.md:108]:
```suggestion
- `.` may not work to open the {% data variables.product.prodname_serverless %} according to your local keyboard layout. In that case, you can open any {% data variables.product.prodname_dotcom %} repository in the {% data variables.product.prodname_serverless %} by changing the URL from `github.com` to `github.dev`.
```

**Review by skedwards88 — COMMENTED** [review]
Thank you for this addition! I made one nit style suggestion. I'll go ahead and incorporate that then get this merged down for you.

**Review by MonsterMar — COMMENTED** [review]
Merged

**Review by MonsterMar — COMMENTED** [review]
`

Since the browser may already have an action registered for those keybindings, we adjust the default for VS Code in the web. | Use the adjusted defaults. They appear on tooltip hovers over the debug actions.Step over is Alt+F10 (instead of F10) on all browsers.Step into in the web is Alt+F11 (instead of F11) on Windows browsers.
-- | --

**Review by MonsterMar — COMMENTED** [review]
bamurtaugh

**Review by ramyaparimi — APPROVED** [review]

**Pull request merged**